### DISCUSSION TITLE
Versions via node/npm

### DISCUSSION THREAD

**Discussion by astridx** [maplibre/maplibre-gl-js#313] [Q&A] [answered]
Can we define the version number that is installed by default via npm? 

Why do I ask? I just used the command `npm i maplibre-gl`. Version `/maplibre-gl@1.14.0-rc.1` was installed.

I had expected that only stable versions would be installed. 

Do someone here know if it is possible to determine the version that is installed when the command  `npm i maplibre-gl`  is entered without a version number 
and
do we want that in this case only a stable version to be installed.

**Comment by wipfli** [comment, marked as answer]
https://github.com/maplibre/maplibre-gl-js/pull/314
